src/plugins/share/server/saved_objects/url.ts

```diff
@@ -37,6 +45,9 @@ export const url: SavedObjectsType = {
       createDate: {
         type: 'date',
       },
+      // Legacy field - contains already pre-formatted final URL.
+      // This is here to support old saved objects that have this field.
+      // TODO: Remove this field and execute a migration to the new format.
```

Comment:
will be possible to migrate to new format? we would need a way to get from an url string to locator params, which might not be doable ?

Reply:
We will not be able to migrate this to dedicated locators, say dashboard locator or discover locator. But we can have a special "url locator" which only has URL in its params.